README.md

```diff
@@ -77,12 +77,14 @@ const img = document.getElementById('img')
 
 // Load model from my S3.
 // See the section hosting the model files on your site.
-nsfwjs.load().then(function (model) {
-  model.classify(img).then(function (predictions) {
+nsfwjs.load()
+  .then(function (model) {
+    return model.classify(img)
+  })
+  .then(function (predictions) {
     // Classify the image
```

Comment:
this comment seems out of place, too.  Not your fault, but could you move it?  Technically, by the time it gets here the image has been classified.

Reply:
Moved
Really overlooked the comment